tests/functional/flakes/common.sh

```diff
@@ -1,10 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
# shellcheck shell=bash
```

Reply:
we don't really run this.

Reply:
I suppose the directive works just as well.